meshtastic/__init__.py

```diff
@@ -190,6 +193,13 @@ def _receiveInfoUpdate(iface, asDict):
     portnums_pb2.PortNum.TEXT_MESSAGE_APP: KnownProtocol(
         "text", onReceive=_onTextReceive
     ),
+    portnums_pb2.PortNum.RANGE_TEST_APP: KnownProtocol(
+        "rangetest", onReceive=_onTextReceive
+    ),
+    portnums_pb2.PortNum.DETECTION_SENSOR_APP: KnownProtocol(
+        "rangetest", onReceive=_onTextReceive
```

Comment:
probably:
"detectionsensor", onReceive=_onTextReceive

Reply:
That would make more sense, huh. Copy-paste error on my part there, haha